src/Tribe/Query.php

```diff
@@ -40,11 +40,10 @@ public static function parse_query( $query ) {
 		}
 
 		// Work out if this is an Event query or not, do not set the flag yet.
-		$is_event_query = (array) $query->get( 'post_type' ) === [ TEC::POSTTYPE ];
-		$any_post_type  = (array) $query->get( 'post_type' ) === [ 'any' ];
-		$is_main_query  = $query->is_main_query();
+		$is_event_query   = (array) $query->get( 'post_type' ) === [ TEC::POSTTYPE ];
```

Comment:
Is there only formatting changes here? I am having trouble seeing if there is any other changes here.

Reply:
Below is the bigger change. Basically all I did was move the block of code into the `if ( $is_main_query ) {` block, instead of being outside below it. You'll notice two closing curly braces at the end of the change vs one.